### PR TITLE
Some cleaning and README update

### PR DESCRIPTION
**Pull request by morucci**
- Remove useless stuff in example
- Update example/config.yaml
- Update README.md
- Add create_base.sh script to help seting up base tree from
  qcow2.
- Make checks related to FS more soft
- Add support for ubuntu 14.04

### CODE REVIEW

File: create_base.sh [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+function clean {
+    mount | grep "/tmp/tree" && sudo umount /tmp/tree
+    losetup | grep $dev && sudo losetup -d $dev || true
+    [ -d /tmp/tree ] && rm -Rf /tmp/tree || true
+    [ -f /tmp/img.raw ] && rm -f /tmp/img.raw || true
+}
+
+function prepare {
+    sudo yum install qemu-img
```
Comment (sbadia) [inline create_base.sh]: Just in order to keep RedHat/Debian compat :-)
```bash
sudo yum install qemu-img kpartx rsync || sudo apt-get install qemu-utils kpartx rsync
```

File: README.md [hunk 2]
```diff
@@ -25,34 +26,26 @@ has to deploy 6 differents virtual machines.
 ## requirements
 
 * lxc
```
Comment (sbadia) [inline README.md]: missing target for the links (markdown syntax) ?

**Comment by sbadia** [comment]
Just some inline comments, otherwise :+1: thanks @morucci !!

File: README.md (hunk 2 above)
Comment (morucci) [inline README.md]: I wouldn't put links here. So I removed the link syntax :)

**Comment by morucci** [comment]
Thx Seb for the review. If no others raise concern I'll merge it today.

**Pull request merged**